enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -566,22 +591,41 @@ def can_redeem(self, request, enterprise_customer_uuid):
                         "reason": reason,
                         "policy_uuids": [policy.uuid for policy in policies],
                     })
+
+            # Select one redeemable policy (i.e. use the "policy resolver") and serialize it for output.
             if redeemable_policies:
                 resolved_policy = SubsidyAccessPolicy.resolve_policy(redeemable_policies)
-                serialized_policy = serializers.SubsidyAccessPolicyRedeemableResponseSerializer(resolved_policy).data
 
+            # Determine if the learner has already redeemed the requested content_key.
             has_successful_redemption = any(
                 redemption['state'] == TransactionStateChoices.COMMITTED
                 for redemption in redemptions
             )
-            can_redeem_for_content_response = {
+
+            # Determine the price for content for display purposes only.
+            try:
+                content_metadata = self.subsidy_client.get_subsidy_content_data(enterprise_customer_uuid, content_key)
+                # Note that the "content_price" key is guaranteed to exist, but the value may be None.  Also, the
+                # upstream field type is a string representing a float, and we passthrough that verbatim.
+                content_price = content_metadata["content_price"]
```

Comment:
FYI, the code as written just passes-through a string representing decimal dollars (which is how the list_price is stored in the content metadata upstream), rather than integer cents used in the rest of this service, including in other API responses.  My personal opinion would be to have this convert the output to be consistent with the rest of the quantities returned by this API, but I also was just referencing the API spec: https://github.com/openedx/enterprise-access/blob/6ad37aac9b3fbdf594c2409fd8b7ad8aa7ab3555/docs/decisions/0006-api-specification-for-enterprise-mfes.rst#L1

Open to suggestions!